### PR TITLE
Footer text

### PR DESCRIPTION
**Pull request by YuriGizra**
#64

selection_001

### CODE REVIEW

File: ibli/themes/custom/bootstrap_subtheme/templates/page.tpl.php [hunk 1]
```diff
@@ -72,12 +72,13 @@
...
  * @ingroup themeable
  */
+
 ?>
 <header id="navbar" role="banner" class="<?php print $navbar_classes; ?>">
   <div class="container">
     <div class="navbar-header">
       <?php if ($logo): ?>
-        <a class="logo navbar-btn pull-left" href="<?php print $front_page; ?>" title="<?php print t('Home'); ?>">
+        <a class="logo naavbar-btn pull-left" href="<?php print $front_page; ?>" title="<?php print t('Home'); ?>">
```
Comment (YuriGizra) [inline ibli/themes/custom/bootstrap_subtheme/templates/page.tpl.php]: Remove one a from the class selector.

**Comment by NofarGizra** [comment]
@YuriGizra 
The address that appears on the footer is not just text in the footer - it is the "Contact Us" address.
I see that the address is also used in the "Contact Us" page, and it shouldn't be duplicated.
I suggest that we use the page element both in the "Contact Us" page and in the footer, so instead of the name "footer text" let's call it "contact address".

File: ibli/themes/custom/bootstrap_subtheme/templates/page.tpl.php [hunk 2]
```diff
@@ -72,6 +72,7 @@
  *
  * @ingroup themeable
  */
+
```
Comment (NofarGizra) [inline ibli/themes/custom/bootstrap_subtheme/templates/page.tpl.php]: Remove space.

**Comment by NofarGizra** [comment]
@YuriGizra 
Please re-roll.

**Comment by NofarGizra** [comment]
Merged.

**Pull request closed without merging**